### PR TITLE
[master] Using disk name to mount instead of UUID

### PR DESCRIPTION
**Pull request by PanYaLian**
The disks for the same image will have the same UUID, it will cause mount
failed when deploy multiple disk with the same image

Signed-off-by: ylpan <ylpan@cn.ibm.com>

### CODE REVIEW

File: zthin-parts/zthin/bin/unpackdiskimage [hunk 1]
```diff
@@ -1258,6 +1261,8 @@ EOF
         fi
     fi
 
+    blockdev --flushbufs ${DEST_DEV} > /dev/null
```
Comment (jichenjc) [inline zthin-parts/zthin/bin/unpackdiskimage:1264]: is this related? seems not mention in your commit message?

Comment (PanYaLian) [reply zthin-parts/zthin/bin/unpackdiskimage:1264]: it is used to sync the data of the disk, not related to the UUID mount failed, but will help to lessen the other data unsync problem
